assets/css/avatar.css

```diff
@@ -1,85 +1,83 @@
-
 .comp-lib-avatar-color {
```

Comment:
![image](https://user-images.githubusercontent.com/91944618/155195630-73e94a40-2020-47de-8ed8-08395062285b.png)
Suggestion readability of the text could be improved.